src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -47,7 +47,7 @@ internal sealed partial class WorkCoordinator
 
                 _listener = listener;
                 _documentTrackingService = _registration.Workspace.Services.GetRequiredService<IDocumentTrackingService>();
-                _workspaceConfigurationService = _registration.Workspace.Services.GetService<IWorkspaceConfigurationService>();
+                _solutionCrawlerOptions = _registration.Workspace.Services.GetService<ISolutionCrawlerOptionsService>();
```

Comment:
Why not have a default implementation at this layer instead of using a nullbale?

Reply:
@genlu Preserving the existing behavior; I don't know why that would have been done with the other service either, but I don't want to find out.